careerfairsystems/nexpo-app
# Update project to make Expo Go work and corrected the fonts/colors for the app

There was a solution to Expo Go issues!
The project needed to update to Expo 46 and renew some deprecated dependencies to make it work on the Expo Go application. However there are some more deprecated dependencies which needs to be fixed but that is a future issue as the app is still working.

To test this you need to do the following:
- Update/Download the latest version of the Expo Go application
- Make sure you are on the same Wifi/Network on the phone as you are on your computer
- Scan the QR code when you have done `npm install` and `npm run start`

Hopefully this works for you as well. Try also with the Backend deployed on your computer and login.

Also there have been some tweaks to the fonts and colors of the app to match the graphical profile of ARKAD better.

Fixes the issues #267 and #262
